test/bigtest/tests/circulation-rules-test.js

```diff
@@ -1054,5 +1054,37 @@ describe('CirculationRules', () => {
     it('should choose loan policy as a fallback', () => {
       expect(savedRules).to.equal(`m 1a54b431-2e4f-452d-9cae-9cee66c9a892 5ee11d91-f7e8-481d-b079-65d708582ccc: l ${lPolicy.id} r ${rPolicy.id} n ${nPolicy.id}`);
     });
+
```

Comment:
It looks like this test case is very similar to the previous one in this file. I seems, they can be combined